spec/spec_helper.rb

```diff
@@ -172,6 +172,14 @@ def refresh_index_without_updating(klass_name)
   $new_elasticsearch.indices.refresh(index: "ao3_test_#{klass_name}s")
 end
 
+def run_all_indexing_jobs
+  %w[main background stats].each do |reindex_type|
+    ScheduledReindexJob.perform reindex_type
+  end
+  # Specs are too fast! Wait for ES...
+  sleep 1
```

Comment:
Instead of introducing a delay, would it work to refresh the indices using `refresh_index_without_updating`?

Reply:
Thank you! I was about to up the delay 😂

Reply:
Glad to help! Fingers crossed that it works. The PR looks good otherwise.